doc/samples/recipes-sample/cmake-hello-world/files/debian/compat

```diff
@@ -0,0 +1 @@
+10
```

Comment:
You don't have to create compat file. ISAR will create it. So, use ``` DEBIAN_COMPAT ``` variable instead of having own compat file. ``` DEBIAN_COMPAT ``` is defined in [conf/distro/debian-bookworm.conf](https://github.com/ilbers/isar/blob/c020ec69b090db0ced128fdd549d0c84acad8781/meta/conf/distro/debian-bookworm.conf) which is include from emlinux-bookworm.conf.


Add ``` DEBIAN_COMPAT ``` to the ``` TEMPLATE_VARS ``` variable which is defined in the cmake-hellow-world_01.bb.

Add following line to the Build-depands setting in files/debian/control.tmpl.
```
debhelper-compat (=${DEBIAN_COMPAT})
```

Reply:
Thanks for the review. I've fix your points, so please confirm it.